src/blocks/team/WithOverlappedCards/WithOverlappedCards.js

```diff
@@ -33,9 +33,8 @@ const mock = [
   },
 ];
 
-const WithOverlappedCards = ({list}) => {
+const WithOverlappedCards = ({list=[]}) => {
```

Comment:
@giseleblair this change, setting a default, fixed the compile error

Reply:
got it, thank you!